modules/tracktion_engine/timestretch/tracktion_RubberBand.cpp

```diff
@@ -0,0 +1,365 @@
+/*
+    ,--.                     ,--.     ,--.  ,--.
+  ,-'  '-.,--.--.,--,--.,---.|  |,-.,-'  '-.`--' ,---. ,--,--,      Copyright 2018
+  '-.  .-'|  .--' ,-.  | .--'|     /'-.  .-',--.| .-. ||      \   Tracktion Software
+    |  |  |  |  \ '-'  \ `--.|  \  \  |  |  |  |' '-' '|  ||  |       Corporation
+    `---' `--'   `--`--'`---'`--'`--' `---' `--' `---' `--''--'    www.tracktion.com
+
+    Tracktion Engine uses a GPL/commercial licence - see LICENCE.md for details.
+*/
+
+#include "tracktion_RubberBand.h"
+
+#ifdef __GNUC__
```

Comment:
These were required to silence warnings on clang on macOS but they might need to be refined for GCC on Linux and will need a different set on Windows